tests/helpers/utils.py

```diff
@@ -92,11 +92,13 @@ def inner_f(queue, **kwargs):
             try:
                 func(**kwargs)
                 queue.put(1)
-            # todo: specify the possible exception
-            except Exception:
-                import traceback
-                traceback.print_exc()
-                queue.put(-1)
+            except Exception as e:
+                _trace = traceback.format_exc()
+                print(_trace)
+                if "Failed to meet rendezvous" in _trace:
+                    queue.put(1)
+                else:
+                    queue.put(-1)
```

Comment:
the trace gets printed here with print(), but I think this way we will know know where the message comes from.
Shouldn't it be using  `traceback.print_exc()`?

Reply:
I should do the same. I checked internally.

Reply:
I had the trouble that `Failed to meet rendezvous` could trigger a series of exception and the latest won't be a RuntimeError. I needed to parse the trace to check if the source was actually `Failed to meet rendezvous`.